cmd/wazero/wazero.go

```diff
@@ -431,6 +472,37 @@ func printRunUsage(stdErr io.Writer, flags *flag.FlagSet) {
 	flags.PrintDefaults()
 }
 
+func startCPUProfile(stdErr io.Writer, path string) (stopCPUProfile func()) {
```

Comment:
wrong rebase?

Reply:
I was missing a rebase indeed, thanks for pointing it out!